{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -1,36 +1,53 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python:
+{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
+  - "{{ env }}"
+{% endfor %}
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
+{% endif %}
   matrix:
-    - TOXENV=check
-{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
-    - TOXENV={{ env }}-cover
+    - COVERAGE=cover
       {%- if cookiecutter.coveralls|lower == 'yes' %}
         {%- if cookiecutter.c_extension_support|lower == "yes" %},extension-coveralls{% endif %},coveralls
       {%- endif %}
       {%- if cookiecutter.codecov|lower == 'yes' %},codecov{% endif %}
-    - TOXENV={{ env }}-nocov
-{% endfor %}
+    - COVERAGE=nocov
+matrix:
+  include:
+    - python: "3.4"
+      env: TOXENV=check
+
 before_install:
-  - python --version
```

Comment:
It's a mess: https://github.com/pypa/pip/issues

I don't trust pip. I don't trust travis. They have bugs and I don't have handy references to give to you :|

Travis may have fixed issues I had but point is, I can't trust them to put the proper stuff in the environment, so I have to printout the versions, in case they mess it up.